examples/tag_filter.c

```diff
@@ -0,0 +1,32 @@
+#include <assert.h>
+#include <stdio.h>
+
+#include "mimpi_err.h"
+#include "../mimpi.h"
+
+int main() {
+    MIMPI_Init(false);
+    int rank = MIMPI_World_rank();
+    int a[1] = {1}, b[1] = {2};
```

Comment:
Czy nie byłoby wygodniej używać jednak pojedynczych `int`ów zamiast tablic?

Reply:
Pewnie by było. Mogę poprawić ale chyba nie ma to większego znaczenia.